packages/notifications/src/elements/Alert.tsx

```diff
@@ -7,17 +7,36 @@
 
 import React, { HTMLAttributes } from 'react';
 import PropTypes from 'prop-types';
-import { StyledAlert, IStyledAlertProps } from '../styled';
-import { ARRAY_VALIDATION_TYPE } from '../utils/types';
+import { StyledAlert, StyledIcon } from '../styled';
+import { validationIcons, validationHues } from '../utils/icons';
+import { ARRAY_VALIDATION_TYPE, VALIDATION_HUE } from '../utils/types';
+import { NotificationsContext } from '../utils/useNotificationsContext';
+
+export interface IAlertProps {
+  type: 'success' | 'warning' | 'error' | 'info';
+}
```

Comment:
Props should have a helpful description comment, particularly because it gets generated into the documentation prop sheet.